omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -92,7 +92,7 @@
       access_by_lua_block { validator.validate("GET") }
       types { }
       default_type text/plain;
-      add_header Content-MD5 <%= Digest::MD5.base64digest(::File.read(node['private_chef']['required_recipe']['path'])) %>;
+      add_header Content-MD5 <%= OpenSSL::Digest::MD5.base64digest(::File.read(node['private_chef']['required_recipe']['path'])) %>;
```

Comment:
Interesting that OpenSSL::Digest::MD5 works in fips mode...

Reply:
It is not. I changed to OpenSSL implementation for consistency - might just revert this (or word the commit message better)